backend/Contexture.Api/Infrastructure/NStoreBased.fs

```diff
@@ -395,7 +395,11 @@ type Storage(persistence: IPersistence, clock: Clock, logger: INStoreLoggerFacto
             return
                 { new Subscription with
                     member _.Name = name
-                    member _.Status = status ()
+                    member _.Status =
+                        if pollingClient.IsActive then
+                            status ()
+                        else
+                            Stopped (Position.from pollingClient.Position)
```

Comment:
`pollingClient.IsActive` should always be `true` otherwise assume unexpected stop most likely caused by exception in `ReadAllAsync` of underlying persistence.

Example of error could be database connection exception when calling `GetContextAsync` that is not delegated to the subscription - subscription.OnError is not called
https://github.com/nstoredev/NStore/blob/bd62cf3fbe1ad1eb3cb4c3fdbf64267350f8c436/src/NStore.Persistence.MsSql/MsSqlPersistence.cs#L28-L46